subprojects/docs/src/snippets/dependencyManagement/customizingResolution-consistentResolution/kotlin/build.gradle.kts

```
@@ -0,0 +1,34 @@
+plugins {
+    `java-library`
+}
+
+repositories {
+    mavenCentral()
+}
+
+// tag::dependencies[]
+dependencies {
+    implementation("org.codehaus.groovy:groovy:3.0.1")
+    runtimeOnly("io.vertx:vertx-lang-groovy:3.9.4")
+}
+// end::dependencies[]
+
+if (project.hasProperty("explicitRule")) {
+    // tag::explicit-configuration[]
+    configurations {
+        runtimeClasspath {
+            shouldResolveConsistentlyWith(compileClasspath.get())
+        }
```

```suggestion
        runtimeClasspath.get().shouldResolveConsistentlyWith(compileClasspath.get())
```

Would be great to use the same notation in Groovy and Kotlin snippet to have the docs not "jump" when you swithc languages. And also to not have it unnecessarily confusing.